src/main/java/graphics/Texture.java

```diff
@@ -17,6 +17,7 @@
 import static org.lwjgl.stb.STBImage.*;
 
 /**
+ * <h1>Azurite</h1>
```

Comment:
These Azurite headers seem unnecessary. If people are reading the documentation, they will already be aware that it is documentation for Azurite.